### PR TITLE
Enhance Documentation for nuxt-mongoose

### PR DESCRIPTION
**Pull request by NormVg**
#### **Summary**
This PR improves the documentation for the `nuxt-mongoose` module by making it more user-friendly and comprehensive. The changes aim to simplify the onboarding process and clarify advanced usage scenarios.

---

#### **Changes Made**
1. **Quick Setup Section**  
   - Added a guide for users who prefer to use the default configuration.  
   - Demonstrated how to provide the MongoDB URI in a `.env` file without modifying the `nuxt.config.ts`.  

2. **`defineMongooseModel` Examples**  
   - Highlighted the importance of the `options.collection` field when connecting to an existing collection.  
   - Provided simplified and generic schema examples for better readability.  

3. **Formatting and Highlights**  
   - Improved section hierarchy and added subheadings to make key points stand out.  
   - Included code comments for better context in examples.  

---

#### **Why These Changes?**
- To reduce ambiguity for new users.  
- To prevent common pitfalls like unintentional collection creation.  
- To improve the overall usability and readability of the documentation.  

---

#### **Testing**
Documentation updates have been reviewed for clarity, correctness, and relevance.  

---

Let me know if any further changes are needed. 😊  

---

### CODE REVIEW

**Comment by arashsheyda** [comment]
thank you for your contribution, looks great!

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-![nuxt-mongoose](https://docs.arashsheyda.me/modules/nuxt-mongoose.jpg)
+![nuxt-mongoose](https://nuxt-mongoose.nuxt.space/cover.jpg)
```
Comment (arashsheyda) [inline README.md:1]: can we just not change this? thanks

Comment (NormVg) [reply README.md:1]: okay ya definitely

**Review by arashsheyda — APPROVED** [review]
looks great, thanks for your contribution!

**Pull request merged**